src/wallet/wallet2.cpp

```diff
@@ -5489,7 +5209,7 @@ bool wallet2::multisig(bool *ready, uint32_t *threshold, uint32_t *total) const
     *ready = !(get_account().get_keys().m_account_address.m_spend_public_key == rct::rct2pk(rct::identity()));
   return true;
 }
-
+//----------------------------------------------------------------------------------------------------
```

Comment:
Since I basically re-wrote the entire multisig address implementation, I decided to do it right and clean up whitespace issues.

- `//----` barriers make it easier to see the begin/end of functions, which aids code navigation
- lambdas shouldn't be unreadable blobs of text (I am not a computer)
- `//---` barriers should be consistent (re: `multisig.cpp`, I made the barriers consistent with the new files I added)